src/server.ts

```diff
@@ -2,86 +2,21 @@ import * as express from 'express';
 import * as path from 'path';
 
 import * as meetbotManager from './meetbot-manager';
+import api from './api';
 
 const HTTP_PORT = process.env.HTTP_PORT || 8080;
-const server = express();
 
+const server = express();
 server.use(express.json());
-server.use('/ui', express.static(path.join(__dirname, './public')));
-
-server.get('/', (_req, res) => {
-	res.send(
-		`Meetbot server is online! Listening for requests on port ${HTTP_PORT}`,
-	);
-});
```

Comment:
@edufschmidt I hope you implemented an alternative for this. What I would expect for this for now is

People open the public URL (`/`) -- gets redirected to --> `/ui` instead

Later when the login flow is complete, 

People open the public URL ('/') --> get redirected to /login instead? What do you think?

Reply:
The UI is now available at `/` (see https://github.com/balena-io-playground/meetbot/blob/efs/dashboard/src/server.ts), as someone suggested during the hack week. Once we have auth in place, unauthenticated users will be redirected to the login page as you described, but this logic will be implemented on the client-side, since all the server does is serve the API and the SPA static bundle.